src/main/java/com/exclamationlabs/connid/base/zoom/adapter/ZoomUsersAdapter.java

```diff
@@ -110,7 +126,61 @@ protected ZoomUser constructModel(
 
     user.setGroupsToRemove(readAssignments(multiValueRemoved, GROUP_IDS));
     user.setGroupsToAdd(readAssignments(multiValueAdded, GROUP_IDS));
+    // Adding Zoom Phone feature.
+    Boolean isZoomPhone =
+        AdapterValueTypeConverter.getSingleAttributeValue(
+            Boolean.class, attributes, ZOOM_PHONE_FEATURE);
+    ZoomFeature feature = new ZoomFeature();
+    feature.setZoomPhone(isZoomPhone);
+    user.setFeature(feature);
+
+    // Adding the Calling Plans
```

Comment:
Do we want to go through the Calling plan and phone field logic below if "feature" is null/false?  Or possibly surround with a large "if" block?  It's fine this way if you feel it's non-breaking ... just wanted to know the intent.  Thanks.

Reply:
The approach supports delta updates. Midpoint my not send the feature as part of the delta. We read the current state to user before performing updates. So decision can be made in the invocator.